client/termsetting/termsetting.ts

```diff
@@ -572,7 +572,11 @@ function setInteractivity(self: TermSettingInstance) {
 		type opt = { label: string; callback: (f?: any) => void }
 		const options: opt[] = []
 
-		if (self.term?.type == 'categorical' && self.q?.groupsetting?.inuse) {
+		if (
+			self.term?.type == 'categorical' &&
+			self.q?.groupsetting?.inuse &&
+			self.opts.usecase?.regressionType != 'logistic'
```

Comment:
the condition should be if self.q.groupsetting.inuse=true and self.q.mode=binary, then do not show the option. this will be effective for any context of q.mode=binary, not just specific to one regressionType and later won't work for a new context that cannot be detected by regressionType

please wait for #692 to complete first